civihr_employee_portal/src/Helpers/TaxonomyHelper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Helpers;
+
+use stdClass;
```

Comment:
this is not necessary

Reply:
Without this I got error in the build process, something like `stdClass' not found in ...`

Reply:
This happens because of the namespace declaration on line 3. To fix the error, change line 49 to `$term = new \stdClass()`;